drivers/web-worker-driver/src/jsCommonMain/kotlin/app/cash/sqldelight/driver/worker/CreateWebWorkerDriver.kt

```diff
@@ -0,0 +1,5 @@
+package app.cash.sqldelight.driver.worker
+
+import app.cash.sqldelight.db.SqlDriver
+
+expect fun createWebWorkerDriver(): SqlDriver
```

Comment:
The web worker driver is designed to support more than just one worker implementation, so this function should be able to accept a `Worker` parameter to allow a consumer to define which worker should be loaded.

Reply:
It is still possible to create WebWorkerDriver with custom `Worker`.
I think we could just rename this method to `createDefaultWebWorkerDriver()` instead.